src/azul/__init__.py

```diff
@@ -524,9 +552,13 @@ class Plugin:
         plugins: Mapping[str, Plugin]
         sources: AbstractSet[str]
 
-        _it_catalog_re: ClassVar[re.Pattern] = re.compile(r'it[\d]+')
+        _catalog_re: ClassVar[str] = r'([a-z][a-z0-9]*(-[a-z0-9]+)*)'
+        _catalog_re: ClassVar[str] = r'(?=.{1,64}$)' + _catalog_re
```

Comment:
```suggestion
        _catalog_re = r'([a-z][a-z0-9]*(-[a-z0-9]+)*)'
        _catalog_re = r'(?=.{1,64}$)' + _catalog_re
```

Only need to set the type hint once per attribute.